src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/Namespace/NewAzureEventHubNamespace.cs

```diff
@@ -92,15 +92,34 @@ public class NewAzureEventHubNamespace : AzureEventHubsCmdletBase
         public Hashtable Tag { get; set; }
 
         /// <summary>
+        /// Indicates whether AutoInflate is enabled.
+        /// </summary>
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Indicates whether AutoInflate is enabled")]
+        public bool? IsAutoInflateEnabled { get; set; }
+
+        /// <summary>
+        /// Upper limit of throughput units when AutoInflate is enabled.
+        /// </summary>
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Upper limit of throughput units when AutoInflate is enabled, vaule should be within 0 to 20 throughput units.")]
```

Comment:
when AutoInflate is disabled and try to set MaximumThroughputUnits is been, RP will return an error